cookiecutter/prompt.py

```diff
@@ -198,41 +201,69 @@ def prompt_for_config(context, no_input=False):
             continue
 
         try:
-            if isinstance(raw, list):
-                # We are dealing with a choice variable
-                val = prompt_choice_for_config(
-                    cookiecutter_dict, env, key, raw, no_input
-                )
-                cookiecutter_dict[key] = val
-            elif not isinstance(raw, dict):
-                # We are dealing with a regular variable
-                val = render_variable(env, raw, cookiecutter_dict)
-
-                if not no_input:
-                    val = read_user_variable(key, val)
-
-                cookiecutter_dict[key] = val
+            cookiecutter_dict = get_cookiecutter_values(
+                key, raw, cookiecutter_dict, meta_information
+            )
         except UndefinedError as err:
             msg = "Unable to render variable '{}'".format(key)
             raise UndefinedVariableInTemplate(msg, err, context)
 
     # Second pass; handle the dictionaries.
     for key, raw in context['cookiecutter'].items():
+        # Check for conditional keywords.
+        if key.startswith(conditional_keyword):
+            conditional_value_to_check = key[conditional_keyword_length:]
+            if cookiecutter_dict[conditional_value_to_check] == 'yes':
```

Comment:
```suggestion
            if cookiecutter_dict[conditional_value_to_check].lower() == 'yes':
```

Just in case we  do `YES` ?